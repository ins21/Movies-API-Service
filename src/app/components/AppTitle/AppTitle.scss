@import "/src/styles/constants/index";

.app-title {
  font-size: 16px;
  color: $primary;

  &__first-word {
    font-weight: 900;
  }

  &__second-word {
    font-weight: 100;
    letter-spacing: 1px;
  }
}
