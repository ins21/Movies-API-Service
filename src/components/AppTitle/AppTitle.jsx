import React from 'react';

export const AppTitle = () => (
  <div className='app-title'>
    <span className='app-title__first-word'>netflix</span>
    <span className='app-title__second-word'>roulette</span>
  </div>
);
