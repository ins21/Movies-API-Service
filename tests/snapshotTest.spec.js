import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import { AppTitle } from '../src/app/components/AppTitle/AppTitle.jsx';

jest.mock('../src/app/components/AppTitle/AppTitle.scss', () => ({}));

configure({ adapter: new Adapter() });

describe('snapshot test of "AppTitle" component', () => {
  it('Must work correctly', () => {
    const wrapper = shallow(<AppTitle />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
