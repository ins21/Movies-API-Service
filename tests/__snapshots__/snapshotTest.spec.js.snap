// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test of "AppTitle" component Must work correctly 1`] = `
<div
  className="app-title"
>
  <span
    className="app-title__first-word"
  >
    netflix
  </span>
  <span
    className="app-title__second-word"
  >
    roulette
  </span>
</div>
`;
